Makefile

```diff
@@ -115,11 +118,12 @@ start: install
 	$(INSTALL_DIR)/bin/python manage.py runserver $(PORT) --settings=tests.settings
 
 # Lint the project
-lint: clean_working_directory
+lint: clean_working_directory $(INSTALL_DIR)/bin/flake8
 	$(call header,"Linting code")
-	@find . -type f -name '*.py' -not -path '$(INSTALL_DIR)/*' -not -path './docs/*' -not -path './build/*' | xargs flake8
+	@find . -type f -name '*.py' -not -path '$(INSTALL_DIR)/*' -not -path './docs/*' -not -path './build/*' | xargs $(INSTALL_DIR)/bin/flake8
+	@. $(INSTALL_DIR)/bin/activate; isort --check-only --recursive $(APP_NAME) tests
```

Comment:
Well, I moved it to virtualenv because right now I always have an error due to not installed flake8 and I don't want to install it globally (e.g. `make lint` doesn't work unless use have flake8). Second, I've been always having a problem with linting because `isort` is not part of it. Not sure what is the problem having these two in virtualenv.